docs/course_materials/linux_intro.html

```diff
@@ -0,0 +1,367 @@
+            <li>Type "terminal" in the search bar</li>
+            <li>Click on the icon named "Terminal"</li>
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+$
+            </code></pre>
+        </section>
+      </section>
+      <section>
+        <section>
+          <h3><b>Getting more info on commands</b></h3>
+          <ul>
+            <li>man</li>
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+$man pwd
+PWD(1)
+
+NAME
+       pwd - print name of current/working directory
+
+SYNOPSIS
+       pwd [OPTION]...
+        </code></pre>
+          <ul>
+            <li>--help</li>
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+$pwd --help
+pwd: pwd [-LP]
+    Print the name of the current working directory.
+
+    Options:
+      -L	print the value of $PWD
+...
+      </code></pre>
+        </section>
+      </section>
+      </section>
+      <section>
+        <section>
+          <h3><b>pwd</b></h3>
+          <ul>
+            <li>Prints the current directory name</li>
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+$pwd
+/home/student
+$
+        </code></pre>
+        </section>
+      </section>
+      <section>
+        <section>
+          <h3><b>mkdir</b></h3>
+          <ul>
+            <li>Creates a directory in the current directory</li>
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+$mkdir my_test
+$
+        </code></pre>
+          <ul>
+            <li>Creates a directory in a specific directory</li>
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+$mkdir /home/student/my_test
+$
+        </code></pre>
+          <ul>
+            <li>Creates a directory and all parent directories if missing</li>
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+$mkdir -p my_test/dir2/dir3/dir4
+$
+        </code></pre>
+        </section>
+      </section>
+      <section>
+        <section>
+          <h3><b>cd</b></h3>
+          <ul>
+            <li>Change to another directory existing in the current directory</li>
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+$cd my_test
+$
+        </code></pre>
+          <ul>
+            <li>Change to a specific directory (Full path)</li>
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+$cd /home/student/my_test
+$
+        </code></pre>
+        </section>
+        <section>
+          <h3><b>cd</b></h3>
+          <ul>
+            <li>Change to user home directory</li>
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+  $cd
+  $
+          </code></pre>
+          <pre><code data-trim data-noescape class="hljs">
+  $cd ~
+  $
+          </code></pre>
+          <ul>
+            <li>Change to parent directory</li>
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+  $cd ..
```

Comment:
maybe adding a pwd before and after will make things more clear, in case someone missed the example on class